shared/js/fb/fb_data_reader.js

```diff
@@ -234,7 +234,7 @@ this.fb = fb;
 
   function doSearchByPhone(number, outRequest) {
     var normalizedNumber = navigator.mozPhoneNumberService.normalize(number);
-    LazyLoader.load(['/shared/js/fb/fb_tel_index.js',
+    window.LazyLoader.load(['/shared/js/fb/fb_tel_index.js',
```

Comment:
the proper fix is add `/* global LazyLoader */` before `'use strict';` , please check other unit test files or jshint document